packages/babel-plugin-strip-runtime/src/index.tsx

```diff
@@ -63,7 +65,8 @@ export default declare<PluginPass>((api) => {
           // We've found something that looks like React.createElement(...)
           // Now we want to check if it's from the Compiled Runtime and if it is - replace with its children.
           const component = path.node.arguments[0];
-          if (!isCCComponent(component)) {
+          // @ts-ignore
```

Comment:
What was the TypeScript error you're suppressing? Could we see if we can fix that instead? 😄

Reply:
Fixed